### DISCUSSION TITLE
Article language

### DISCUSSION THREAD

**Discussion by ajmas** [forem/forem#18692] [Feature Requests]
Recently  being presented with an article in Chinese, which I can’t read, made me wonder whether there was any language indicator when writing an article. There doesn’t seem to be one from what I can see, unless I missed it?

Would there be any interest in adding a language indicator field, with the value defaulting to English or the user’s display language.  If there is, I’d be tempted to contribute the necessary code.

**Comment by yheuhtozr** [comment]
FYI: past attempt https://github.com/forem/forem/pull/14564

**Reply by ajmas** [reply]
Thanks. Reading through the links it isn’t really clear what the  status of the work is.
